### PR TITLE
Improve information consistency

### PR DESCRIPTION
**Pull request by diegonvs**
Hey @eduardoallegrini and @marcoscv-work, what do you think about it guys? I took the WIP of Edu and wrote in a more generic way for creating tabs on Mdx files. 

WIP of https://issues.liferay.com/browse/IFI-790

### CODE REVIEW

File: next.clayui.com/content/docs/components/alerts.mdx [hunk 1]
```diff
@@ -4,6 +4,7 @@ description: "Alerts are used to capture the attention of the user in an intrusi
 packageNpm: "@clayui/alert"
 packageVersion: "3.0.0-milestone.1"
 packageStatus: "Beta"
+sibling: "docs/css/components/alerts.html"
```
Comment (matuzalemsteles) [inline next.clayui.com/content/docs/components/alerts.mdx]: Hey @diegonvs why didn't you add this to `.md` files?

File: next.clayui.com/gatsby/createPages.js [hunk 2]
```diff
@@ -71,6 +72,8 @@ const createDocs = (actions, edges, mdx) => {
```
Comment (matuzalemsteles) [inline next.clayui.com/gatsby/createPages.js]: If adding `sibling` to the `.md` files, we can skip creating the `.mdx` pages here.

**Comment by eduardoallegrini** [comment]
Hi @diegonvs , I think it's great, in this way we have more control over it and we can decide what to link.
My only concern is about the css-pages, should we remove it or keep it to create a bidirectional navigation (components then type and type then components).
If we keep it then we would have a duplicated content (SEO ranking issues, but it shouldn't matter in this phase) in this case how the search works? it's going to "returns" the react or the css page? Maybe we should also assign the template-components to the css-page and invert the graphql query in order to obtain a similar result in the css-pages but is it really necessary?

**Comment by marcoscv-work** [comment]
Hey guys,

About the tabs, it is exactly what we wanted and looks great!
The section: CSS / Components should be not visible, we do not need duplicated content

About the section link list, in this PR we have lost the floating effect:
image

https://next.clayui.com/docs/components/date-picker.html

**Comment by jbalsas** [comment]
Hey @diegonvs, this looks great and good timing for our upcoming milestone! :)

I would simply:
- Remove the `Components` section under CSS. We can just hide it for now if unsure all the content is transferred.
- Rename the `React Components` section to simply `Components`
- [Optional] Can we for now make the `Getting Started` section our main page without the js redirect? We will get back the landing once we go live, but for now, it should be better to get to the content first.

**Comment by diegonvs** [comment]
> The section: CSS / Components should be not visible, we do not need duplicated content
> My only concern is about the css-pages, should we remove it or keep it to create bidirectional navigation (components then type and type then components).

I'm working harder to have a smarter way to remove things duplicated. You guys have no idea 😅 

> Remove the Components section under CSS. We can just hide it for now if unsure all the content is transferred.
> Rename the React Components section to simply Components

It's on my mind. When finishing tabs problem I gonna adjust these things.

> [Optional] Can we, for now, make the Getting Started section our main page without the js redirect? We will get back the landing once we go live, but for now, it should be better to get to the content first.

Honestly, I preferred it as it was before. More information on what Clay is before proceeding to the components. However, if you guys are minding that I can take it.

Thanks for the responses @eduardoallegrini @marcoscv-work @jbalsas ❤️

**Comment by jbalsas** [comment]
Hey @diegonvs 

> Honestly, I preferred it as it was before. More information on what Clay is before proceeding to the components. I gonna do that!

Me too, we'll get it back once we get closer to a final release and can focus back on communication and community. For now, let's focus on our developer docs and information... I've already received some very nice feedback about what we have so far, so let's keep pushing! :)

**Comment by diegonvs** [comment]
Hey @marcoscv-work, for the link list/summary component, do you have any styles in your mind to enhance contrast?

**Comment by diegonvs** [comment]
Ready for review folks!

File: next.clayui.com/src/templates/components.js [hunk 3]
```diff
@@ -0,0 +1,341 @@
+/**
```
Comment (matuzalemsteles) [inline next.clayui.com/src/templates/components.js]: Later we can remove this template and use `docs`. It is not necessary to have this.

**Pull request merged**